Project-2/README.md

```diff
@@ -35,15 +60,76 @@ Tip #4: Work together as a team.
+ ∀r.C = all r-fillers ys of all x instantiate C
+ 
   (b) ∃r-.C and ∀r-.C
+   ∃r-.C = all xs are r-inverse-related to y and falls under concept C
+   ∀r-.C = all xs are r-inverse-related to all ys and this thing falls under concept C
+   
   (c) <=nr and <=nr.C
+  <=nr = role r connects all the xs to no more than n elements
+  <=nr.C = role r connects all the xs to no more than n elements, and they fall under concept C
+  
   (d) ∃r-.C and ∃r-.{a} 
+  ∃r-.C = for all xs, there is at least a thing y, which is r-related to it, and which falls under concept C
+  ∃r-.{a} = for all xs, there is element a, which is r-related to it
 ```
 
 7. There is a delightfully helpful subreddit called "ELI5" which stands for something like "explain it like I'm 5" where users post conceptually challenging questions and other users attempt to provide explanations in simple, jargon-free, terms that presumably a 5 year-old could understand. Using this as a model, explain the _finite model property_. Be sure to provide a simple example and explain when the property might be important, and when it is not so important. 
 
+Definition 3.10. The interpretation I is a model of a concept C with respect to a TBox T if I is a model of T such that CI = ∅. We call this model finite if ΔI is finite. (Baader, 57)
+
+In this scenario, I am the five-year-old that needs someone to explain this concept to.  The answered proposed on the collaborative google doc are extremely helpful.  I don't see a need to try to do it better in this case.  Jeiming's answer:
+
+Bob loves playing games, and he hopes there is a game that can be played with different results forever. 
+One day, his father gives him a game, called Color-card: there are a number of color cards in a box. Each card has a different color, but it is fully colored on one side, while on the other side it is a blank sheet with the colored dot. Once a time, the player chooses any number of cards from the box, and then uses any number of cards he chooses to make combinations. For example, the first time, Bob chooses five color cards from the box, red, yellow, blue, green, brown, then he uses three cards, and put them in such a order: 1, red (front side), 2, blue (back side), 3, green (back side). That is one combination. The player continues playing until there is a combination that repeats. If the player cannot continue playing, he fails. However, if the player fails at one time, he can choose more or/and different cards, then play the next time. The player wins if the cards he chooses can allow him to play forever. 
+Unfortunately, Bob cannot win the game, because no matter how many cards he chooses (the number of cards is not limitless), and how hard he tries, he cannot continue playing without a repeated combination occurring.  
+We say that the Color-card game has finite model property. 
+Finite model property is important because it can be used to design a decidable algorithm for the satisfiability of its concepts. 
```

Comment:
Ah but this isn't quite helpful because the color-game is only given as an example of a system with the finite model property. What features of the game underwrite the game exhibiting that property? If a 5 year old says 'what's a card game' and I describe solitaire, then say that's a card game, they might be inclined to think poker isn't a card game, since it's so different from solitaire.